api-schema/src/main/java/org/stellar/anchor/api/asset/StellarAssetInfo.java

```diff
@@ -2,13 +2,91 @@
 
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
-import org.stellar.anchor.api.sep.AssetInfo;
+import org.stellar.anchor.api.sep.operation.Sep31Info;
+import org.stellar.anchor.api.sep.operation.Sep38Info;
+import org.stellar.anchor.api.sep.sep31.Sep31InfoResponse;
+import org.stellar.anchor.api.sep.sep38.InfoResponse;
 
 @Data
-public class StellarAssetInfo extends AssetInfo {
-
-  String issuer;
+public class StellarAssetInfo implements AssetInfo {
+  String id;
 
   @SerializedName("distribution_account")
   String distributionAccount;
+
+  @SerializedName("significant_decimals")
+  Integer significantDecimals;
+
+  DepositWithdrawInfo sep6;
+  DepositWithdrawInfo sep24;
+  Sep31Info sep31;
```

Comment:
nit: Can we call `Sep31Info` something like `SendInfo`  for consistency? Same with `Sep38Info`.

Reply:
Hmm I thought about it name after `Sep6AndSep24Info` but that sounds so weird...I would like to keep the naming match the config as much as possible

Reply:
I think that's actually better than `SendInfo. ' Let's call it that for now?

Reply:
Sure it will be named `ReceiveInfo` tho to match the config